log-processing/README.md

```diff
@@ -0,0 +1,60 @@
+
+## Solution: Distributed Log Orchestration with Bacalhau
+Rather than forwarding all logs to a monolithic platform, Bacalhau allows you to:
+ - Deploy **daemon jobs** that continuously collect, filter, and compress logs right where they’re generated
+ - Send aggregated data to real-time analytics platforms for quick insights
+ - Store full, raw logs in inexpensive storage (e.g., S3) for compliance, deep-dive, and batch analytics
+ - Query logs directly on compute nodes for immediate troubleshooting
+ - Scale quickly on multiple nodes, either on-prem or across different clouds, with minimal manual intervention
+
+**Key Advantages**:
+ - Substantial bandwidth and cost savings
+ - Flexible ingestion pipelines — use your favorite logging stack or data warehouse
+ - Simplified troubleshooting with local or near-local query options
+ - Straightforward job orchestration via Bacalhau’s CLI or APIs
+
+## Run This Example
+
+### Prerequisites
+- Docker and Docker Compose
+- [Expanso Cloud](https://cloud.expanso.io) account
+
+### 1. Create an Orchestrator
+Use [Expanso Cloud](https://www.cloud.expanso.io/) to create a new Bacalhau network (a managed orchestrator node). Name it however you like.
+
+### 2. Spin Up and Connect Compute Node
+1. Under _"+ Node"_ in your new network, obtain the token and NATS URL from the platform.
+2. In this directory, update `compute.yaml` with the relevant token and NATS URL replacing all instances of `<TODO>`. If necessary, change the number of compute nodes you want to run under `services.compute.deploy.replicas` in `docker-compose.yml`.
+
+3. Start the environment:
+   ```bash
+   docker compose up
+   ```
+   This command will spin up Bacalhau compute nodes, a storage (MinIO) node and OpenSearch nodes.
+
+### 3. Generate Logs
+In the Expanso Cloud dashboard, go to **Network > _YourNetwork_ > Jobs**, create a new job from file, use `generate-logs.yaml` and run it. This [daemon](https://docs.bacalhau.org/cli-api/specifications/job/type#daemon-jobs) job will run on all compute nodes and generate logs for this example. Once submitted you should be redirected to the Job page. Here you can see its status, history and executions. Under the executions tab you should see one execution per node.
+
+### 4. Run the Logstash Job
+In Expanso Cloud, go to **Network > _YourNetwork_ > Jobs**, create or upload the job `logstash.yaml` and run it. This is another [daemon](https://docs.bacalhau.org/cli-api/specifications/job/type#daemon-jobs) job that will collect logs, writing aggregated logs to OpenSearch, and storing raw logs in MinIO. After a few minutes, verify logs are being collected by checking your logs bucket at `localhost:9001` (username and password configured in `docker-compose.yml`) and your OpenSearch dashboard at `localhost:5601`.
+
+### 5. Query Logs directly on Compute Node
```

Comment:
We are missing steps on how to view the query results. For ops jobs like this one we just want to see the query result and not necessary persist it in S3. I think just querying the job logs should be enough, either by directing them to the page in expanso cloud or `bacalhau job logs <id>`